src/components/CommentsList.jsx

```diff
@@ -0,0 +1,32 @@
+import { useState, useEffect } from "react";
+import { getCommentsByArticleID } from "../utils/api";
+import CommentCard from "./CommentCard";
+
+const CommentsList = ({ article_id }) => {
+  const [commentsList, setCommentsList] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState(false);
+
+  useEffect(() => {
+    getCommentsByArticleID(article_id).then(({ comments }) => {
+      setCommentsList(comments);
+      setIsLoading(false);
+    });
+  }, [article_id]);
+
+  if (hasError) return <p className="errorMsg"> No Comments Found. </p>;
+  if (isLoading) return <p className="loadingMsg">Fetching Data...</p>;
+
+  return (
+    <div className="comment-section">
+      <h2>Comment Section</h2>
+      <ul className="comments-list">
+        {commentsList.map((comment) => {
+          return <CommentCard key={comment.comment_id} comment={comment} />;
```

Comment:
I know all of them have comments, but it could benefit from a little paragraph for when an article doesnt have any comments. 
Just for improved UX and to let them know that there aren't any yet. 

Will be more useful when you get to deleting comments later on